kumo-core/src/main/java/com/kennycason/kumo/bg/PixelBoundaryBackground.java

```diff
@@ -32,6 +32,18 @@ public PixelBoundaryBackground(final InputStream imageInputStream) throws IOExce
         final BufferedImage bufferedImage = ImageIO.read(imageInputStream);
         this.collisionRaster = new CollisionRaster(bufferedImage);
     }
+    /**
+     * Creates a PixelBoundaryBackground using an InputStream to load an image
+     *
+     * @param imageInputStream InputStream containing an image file
+     * @param alpha The transparent index, should range from 0 to 10
+     * @throws IOException when fails to open file stream
+     */
+    public PixelBoundaryBackground(final InputStream imageInputStream, int alpha) throws Exception {
```

Comment:
I think alpha should just be a 0-255 value instead of clamping it between [0,10].